server/test/handlers/availability.test.js

```diff
@@ -4,40 +4,197 @@ describe('testing fetching of worker availability functionality', () => {
+        const startTime = '8:00:00';
+        const endTime = '8:30:00';
+
+        // Act
+        const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+        // Assert
+        expect(availableTimes).toBe(null);
+    });
+
+    //TEST 3
+
+    test('rejection of null value for required SchoolId parameter', async () => {
+        // Arrange
+        const workerId = 8000000;
+        const schoolId = null;
+        const startTime = '8:00:00';
+        const endTime = '8:30:00';
+
+        // Act
+        const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+        // Assert
+        expect(availableTimes).toBe(null);
+    });
+
+//TEST 4
+
+test('rejection of null value for required startTime parameter', async () => {
+    // Arrange
+    const workerId = 8000000;
+    const schoolId = 1;
+    const startTime = null;
+    const endTime = '8:30:00';
+
+    // Act
+    const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+    // Assert
+    expect(availableTimes).toBe(null);
+});
+
+//TEST 5
+
+test('rejection of null value for required endTime parameter', async () => {
+    // Arrange
+    const workerId = 8000000;
+    const schoolId = 1;
+    const startTime = '8:00:00';
+    const endTime = null;
+
+    // Act
+    const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+    // Assert
+    expect(availableTimes).toBe(null);
+});
+
+//TEST 6
+
+test('rejection of empty string value for required workerId parameter', async () => {
+    // Arrange
+    const workerId = '';
+    const schoolId = 1;
+    const startTime = '8:00:00';
+    const endTime = '8:30:00';
+
+    // Act
+    const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+    // Assert
+    expect(availableTimes).toBe(null);
+});
+
+//TEST 7
+test('rejection of empty string value for required schoolId parameter', async () => {
+    // Arrange
+    const workerId = 8000000;
+    const schoolId = '';
+    const startTime = '8:00:00';
+    const endTime = '8:30:00';
+
+    // Act
+    const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+    // Assert
+    expect(availableTimes).toBe(null);
+});
+//TEST 8
+
+test('rejection of empty string value for required startTime parameter', async () => {
+    // Arrange
+    const workerId = 8000000;
+    const schoolId = 1;
+    const startTime = '';
+    const endTime = '8:30:00';
+
+    // Act
+    const availableTimes = await availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);
+
+    // Assert
+    expect(availableTimes).toBe(null);
+});
+//TEST 9
+test('rejection of empty string value for required endTime parameter', async () => {
```

Comment:
Hi Nancy,
If possible, I would recommend right clicking on this file and using the Format feature on Visual Studio so the code is easier to read.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've reformatted the server/test/handlers/availability.test.js

Please confirm if this takes care of your suggestion!